modules/sdk-coin-sui/test/unit/utils.ts

```diff
@@ -0,0 +1,33 @@
+import should from 'should';
+import * as testData from '../resources/sui';
+import utils from '../../src/lib/utils';
+
+describe('Sui util library', function () {
+  describe('isValidAddress', function () {
+    it('should succeed to validate valid address', function () {
+      for (const address of testData.addresses.validAddresses) {
+        should.equal(utils.isValidAddress(address), true);
+      }
+    });
+    it('should fail to validate invalid addresses', function () {
+      for (const address of testData.addresses.invalidAddresses) {
+        should.doesNotThrow(() => utils.isValidAddress(address));
+        should.equal(utils.isValidAddress(address), false);
+      }
+      // @ts-expect-error Testing for missing param, should not throw an error
+      should.doesNotThrow(() => utils.isValidAddress(undefined));
+      // @ts-expect-error Testing for missing param, should return false
+      should.equal(utils.isValidAddress(undefined), false);
```

Comment:
is it sufficient to do:
`utils.isValidAddress(undefined).should.be.false()`

maybe able to avoid the `@ts-expect-error` overrides this way

Reply:
doing `utils.isValidAddress(undefined).should.be.false()` still requires `@ts-expect-error`

Reply:
🤔 weird...